llvm/lib/DWARFLinkerParallel/DWARFLinkerCompileUnit.cpp

```diff
@@ -167,6 +167,21 @@ void CompileUnit::cleanupDataAfterClonning() {
   getOrigUnit().clear();
 }
 
+/// Make a best effort to guess the
+/// Xcode.app/Contents/Developer/Toolchains/ path from an SDK path.
+static SmallString<128> guessToolChainBaseDir(StringRef SysRoot) {
```

Comment:
Is there unify this function since it exists both in `DWARFLinker.cpp` and here?

Reply:
Yes, see the discussion above. That's out-of-scope for this patch though.